qa/L0_infer_zero/test.sh

```diff
@@ -55,6 +55,10 @@ rm -fr models && mkdir models
 cp -r /data/inferenceserver/${REPO_VERSION}/qa_identity_model_repository/* models/. && \
     cp -r /data/inferenceserver/${REPO_VERSION}/qa_ensemble_model_repository/qa_identity_model_repository/* models/.
 
+# Remove version-compatible TensorRT models, as they require version-compatibility
+# mode to be turned on when starting the server.
+rm -r models/plan_compatible*
```

Comment:
How difficult would it be for compatible plans to be in a separate folder instead, and explicitly pulled in for the compatibility test? ex:
```
qa_trt_compatible_model_repository
```

Reply:
I think it's more prone to unexpected errors when a model requiring server-specific flags `--backend-config ... compatible=true` is included with the rest of the common QA models.

Reply:
We could. I didn't create one because there is only one version-compatible model, so I didn't love the idea of creating another directory just for this one model. And I couldn't find another directory that seemed to make more sense for this model.

Most of the tests seem to just copy the identity models they need rather than the whole repository. If you think there's a clean, worthwhile way to separate out the compatible model, let me know. If you think it'd be the best option, I could just create that separate directory, since we have ones for other things like the trt_plugin one, but I'd like to ultimately find a way to ensure that high-level model directory doesn't grow in an unorderly way.

Reply:
I think this is fine for now, but if it comes up again I'd probably move it to a separate folder.

Reply:
Regarding high level directory growth, we can always start grouping things like `trt_model_repo` containing `trt_plugin`, `trt_version_compatible`, etc.

Reply:
That's a good idea! I'll try to keep an eye out for this, we might want to start shifting towards that approach as we create more backend-specific repos. Thanks, Ryan.